verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -307,6 +379,62 @@ VName KytheFactsExtractor::ExtractClassInstances(
   return class_instance_vname;
 }
 
+VName KytheFactsExtractor::ExtractPackageImport(
+    const IndexingFactNode& import_fact_node) {
+  const auto& anchors = import_fact_node.Value().Anchors();
+  const Anchor& package_name = anchors[0];
+
+  const VName package_vname(file_path_,
+                            CreatePackageSignature(package_name.Value()));
+  const VName package_anchor = PrintAnchorVName(package_name);
+
+  GenerateEdgeString(package_anchor, kEdgeRefImports, package_vname);
+
+  // case of import pkg::my_variable.
+  if (anchors.size() > 1) {
+    const Anchor& imported_item_name = anchors[1];
+    const VName& defintion_vname =
+        *ABSL_DIE_IF_NULL(SearchForDefinitionVNameInPackage(
+            package_name.Value(), imported_item_name.Value()));
+
+    const VName imported_item_anchor = PrintAnchorVName(imported_item_name);
+    GenerateEdgeString(imported_item_anchor, kEdgeRef, defintion_vname);
+
+    // Add the found definition to the current scope as if it was declared in
+    // our scope so that it can be caputred without "::".
+    scope_context_.top().push_back(defintion_vname);
+  }
+  // case of import pkg::*.
+  else {
+    // Add all the definitions in that package to the current scope as if it was
+    // declared in our scope so that it can be caputred without "::".
+    const auto current_package_scope = package_scope_context_.find(
+        CreatePackageSignature(package_name.Value()));
+    for (const VName& vname : current_package_scope->second) {
+      scope_context_.top().push_back(vname);
+    }
+  }
+
+  return package_vname;
+}
+
+const VName* KytheFactsExtractor::SearchForDefinitionVNameInPackage(
+    std::string package_name, std::string reference_name) {
```

Comment:
I think these parameters can be `absl::string_view`.

And it looks (at-a-glance) like this method could be `const`.